proxylib/proxylib/instance.go

```diff
@@ -183,7 +183,7 @@ func (ins *Instance) PolicyUpdate(resp *envoy_service_discovery.DiscoveryRespons
 			return fmt.Errorf("NPDS: Policy has no endpoint_ips")
 		}
 		for _, ip := range ips {
-			logrus.Infof("NPDS: Endpoint IP: %s", ip)
+			logrus.Debugf("NPDS: Endpoint IP: %s", ip)
```

Comment:
this is un-related to this PR, we can just have one single log entry to print all ips together, something like below

```go
logrus.Infof("NPDS: Endpoint IP: %v", strings.Join(ips, ","))
```

or just

```go
	logrus.Debugf("NPDS: Endpoint IP: %v", ips)
```